### ISSUE TITLE
Almost impossible to know selection in the references window

### ISSUE THREAD

**Issue by ronakg** [sainnhe/gruvbox-material-vscode#23]
Hi,

The selected entry in the references window has no contrast compared to other entries. It's almost impossible to figure out what's selected. 

Example:

Here, the line that starts with `Uses` is selected (middle of the right side window), not the highlighted line that starts with `be {@` (top of the right side window).

<img width="1419" alt="Screen Shot 2020-06-07 at 10 54 07 PM" src="https://user-images.githubusercontent.com/1058322/83996990-3dd38800-a912-11ea-848f-3fd881ffd30b.png">

**Comment by Z4RX** [comment]
Hello!
I think that you have some customisations in your JSON settings.
This theme with no recolors looks like this.
![image](https://user-images.githubusercontent.com/58516282/84069440-31fbca80-a9d3-11ea-8568-ae8c8e7a6f83.png)
So first of all check pls section "workbench.colorCustomizations": { ... }, in your settings file.

**Comment by ronakg** [comment]
Only customization I have is this - 

```json
    "workbench.colorCustomizations": {
        "tab.activeBorder": "#ff0000"
    },
```

It highlights the active tab.

I changed the workbench palette to material, which improves the contrast in the references window, but I like the `flat` palette much more for the workbench.

**Comment by Z4RX** [comment]
My bad, I didn't notice it

**Comment by Z4RX** [comment]
Hello again.
I am just a maintainer so I can not change theme's colors.
This temporary solution can help you.
Add this to your settings JSON to `"workbench.colorCustomizations": { }` section:
`"peekViewResult.matchHighlightBackground": "#b8565180",
 "peekViewResult.selectionBackground": "#72966c50",
 "peekViewEditor.matchHighlightBackground": "#b8565180",`

Example of its change is on the screenshot below.
![image](https://user-images.githubusercontent.com/58516282/84120284-46c17800-aa3e-11ea-92f4-c2ef99c10cdc.png)

**Comment by sainnhe** [comment]
Sorry for late reply, this should be fixed via 5027c7b.

See the line in the middle of this screenshot:

![demo3](https://user-images.githubusercontent.com/37491630/86114428-192b9400-bafd-11ea-9aa1-86a5d7a6d200.png)

**Comment by ronakg** [comment]
It is still really difficult to see what's selected because the selection doesn't have any background color. Just bright foreground is not enough to distinguish within a large list of results.

<img width="806" alt="Screen Shot 2020-06-30 at 10 40 27 AM" src="https://user-images.githubusercontent.com/1058322/86158946-6a686300-babe-11ea-878b-2bf5277a8d91.png">

This is how the default dark theme does it, very easy to figure out what's selected.

<img width="806" alt="Screen Shot 2020-06-30 at 10 40 58 AM" src="https://user-images.githubusercontent.com/1058322/86158982-79e7ac00-babe-11ea-8b09-1348e7d97b3a.png">

**Comment by sainnhe** [comment]
The json key of the background color of peek hover items is the same as the background color of the sidebar hover items(e.g. the file explorer in the left side). Therefore, if I change the bg of peek hover items, the bg of explorer hover items will also be changed. But I'm currently very satisfied with the design of the sidebar.

So very sorry, this is the only fix I can do, it's impossible to change the bg of peek hover items while keeping the bg of sidebar hover items unchanged, unless vscode adds a new json key to specifically control the bg of the peek hover items.

**Comment by ronakg** [comment]
I see what you mean. Can you tell me what I can add to my customizations to have a very subtle grey background for the hover element?

**Comment by sainnhe** [comment]
Try something like this:

```json
    "list.hoverBackground": "#3c3836",
```

See [/src/palette](https://github.com/sainnhe/gruvbox-material-vscode/tree/master/src/palette) for the reference for all colors.

**Comment by ronakg** [comment]
I added the following to my customizations for testing. But that doesn't seem to affect the references window selection background.

<img width="302" alt="Screen Shot 2020-06-30 at 3 16 43 PM" src="https://user-images.githubusercontent.com/1058322/86182508-fa210800-bae4-11ea-9bfe-83725c6132b2.png">

Sidebar change:
<img width="809" alt="Screen Shot 2020-06-30 at 3 16 37 PM" src="https://user-images.githubusercontent.com/1058322/86182563-1624a980-bae5-11ea-9aca-1d97087ea42c.png">


References window:

<img width="316" alt="Screen Shot 2020-06-30 at 3 16 31 PM" src="https://user-images.githubusercontent.com/1058322/86182541-0b6a1480-bae5-11ea-9b39-b91964260f19.png">

**Comment by sainnhe** [comment]
That's strange, it works for me, by using both hovering and keyboard:

![demo4](https://user-images.githubusercontent.com/37491630/86182769-27140000-bb63-11ea-9eaa-7afdadee74ea.png)

Maybe we can open an issue in vscode?

**Comment by ronakg** [comment]
Yes, strange.

I'm able to get all 3 kinds of selection show up in this window.

<img width="809" alt="Screen Shot 2020-06-30 at 3 25 11 PM" src="https://user-images.githubusercontent.com/1058322/86183095-1bcebf00-bae6-11ea-8876-378d93f6a041.png">

The first line with green background is the last item that was expanded (totally useless in the context).
The red highlight is when I hover the mouse over the line.
The last line with bright foreground is the line I selected using keyboard arrow keys (so not specific to vim plugin).

**Comment by sainnhe** [comment]
I see, there is also 3 kinds of selection in my vscode, I'm going to debug it.

**Comment by sainnhe** [comment]
So there is another key named `list.focusBackground` controls the background color:

```json
    "list.hoverBackground": "#ff0000",
    "list.focusBackground": "#00ff00",
```

![demo5](https://user-images.githubusercontent.com/37491630/86184093-ff726700-bb65-11ea-97dd-7e8ffa928660.png)

**Comment by sainnhe** [comment]
This should be fixed via a8daf34, you can update this extension and get it a try.

**Comment by ronakg** [comment]
This is perfect. Thanks!
